### DISCUSSION TITLE
New release?

### DISCUSSION THREAD

**Discussion by shiosai** [tchiotludo/akhq#894] [General]
Given all the new features added since 0.18 - is there a roadmap regarding a new release already?

Thanks!

**Comment by tchiotludo** [comment]
It will be ... :drum: :drum: :drum: 
Ooops, just happen : https://github.com/tchiotludo/akhq/releases/tag/0.19.0 :tada:

**Reply by shiosai** [reply]
Ooops indeed! Thank you :+1:
